src/client/routers/AppRouter.js

```diff
@@ -2,15 +2,35 @@ import React from 'react'
 import { BrowserRouter, Route, Switch, Link, NavLink } from 'react-router-dom'
 import Headers from '../components/Header'
 import MainContent from '../components/MainContent'
-import QuestionPage from '../components/QuestionPage'
+import QuestionOne from '../views/QuestionOne'
```

Comment:
Would it be better to dynamically create these pages instead?
Something like a base template that changes based on what it's fed.

Reply:
yep agreed like a path variable